src/columns.rs

```diff
@@ -94,7 +94,9 @@ where
             match wrapped_lines.get(line_no + column_no * lines_per_column) {
                 Some(column_line) => {
                     line.push_str(column_line);
-                    line.push_str(&" ".repeat(column_width - display_width(column_line)));
+                    line.push_str(
+                        &" ".repeat(column_width - display_width(column_line, options.tab_width)),
```

Comment:
A bit of a side comment: it just occurred to me that it might be nice to do a 

```rust
let display_width = |text| core::display_width(text, options.tab_width);
```

early on in the function. Then everything can stay the same. I'm always on the fence with such things: I don't like indirection and extra layers, but here it might be nice since it cuts down on the repetition.

Reply:
Good suggestion, I think that does make things cleaner. I've made the change in 08e4d4a15c47a9093ef8e2afcb0bf2b0528941a1.

Reply:
Nice!